qa/system-indices/src/main/java/org/elasticsearch/system/indices/SystemIndicesQA.java

```diff
@@ -73,15 +74,15 @@ public Collection<SystemIndexDescriptor> getSystemIndexDescriptors(Settings sett
                         .put(IndexMetadata.SETTING_AUTO_EXPAND_REPLICAS, "0-1")
                         .build()
                 )
-                .setOrigin("net-new")
```

Comment:
Fake origins don't work anymore, since we actually switch the security. I picked tasks since it's not in a module.